File: kotlin/PlanesAndroid/app/src/test/java/OrientationTest.java
import com.google.common.truth.Truth;
import com.planes.common.Orientation;

import org.junit.Test;
import static com.google.common.truth.Truth.assertThat;

public class OrientationTest {

    @Test
    public void Orientation_Values() {
        assertThat(Orientation.values()[0] == Orientation.NorthSouth).isTrue();
        assertThat(Orientation.values()[1] == Orientation.SouthNorth).isTrue();
        assertThat(Orientation.values()[2] == Orientation.WestEast).isTrue();
        assertThat(Orientation.values()[3] == Orientation.EastWest).isTrue();
    }
}

File: kotlin/PlanesAndroid/app/src/test/java/OrientationTest.kt
import com.google.common.truth.Truth
import com.planes.common.Orientation
import org.junit.Test

class OrientationTest {
    @Test
    fun Orientation_Values() {
        Truth.assertThat(Orientation.values()[0] === Orientation.NorthSouth).isTrue()
        Truth.assertThat(Orientation.values()[1] === Orientation.SouthNorth).isTrue()
        Truth.assertThat(Orientation.values()[2] === Orientation.WestEast).isTrue()
        Truth.assertThat(Orientation.values()[3] === Orientation.EastWest).isTrue()
    }
}